### ISSUE TITLE
`rustup update` help is somewhat inaccurate

### ISSUE THREAD

**Issue by futile** [rust-lang-nursery/rustup.rs#528]
I just ran `rustup update`, and it only updated the nightly channel, which I found confusing, since `rustup help update` says:

```
With no toolchain specified, the `update` command updates each of the
stable, beta, and nightly toolchains from the official release
channels, then updates rustup itself.
```

In my case, I didn't have stable and beta installed before, which is (probably) why rustup didn't try 'updating' them. However, the help makes it sound like `rustup update` is similar to `rustup update stable && rustup update beta && rustup update nightly && rustup self update`. Indeed, running `rustup update stable` also installs the stable toolchain.

Only updating the installed toolchains definitely makes sense and seems to be the correct behavior for this case, but the help does not mention this behavior. I actually thought it was a bug that rustup only updated the nightly channel after I read the help. It should say something like `... updates stable, beta and nightly (if they were already installed before) ...` to clear up this confusion.

**Comment by brson** [comment]
Yep you are right. Maybe 'updates any installed toolchains' would be a simple way to explain it. That's again not entirely accurate because some toolchains can't be updated, but it covers most cases newbies will hit.

**Comment by natemara** [comment]
Can this be closed now that #552 is merged?

**Comment by brson** [comment]
Yep. Thanks @natemara !
